build.sbt

```diff
@@ -42,6 +42,19 @@ lazy val core = projectMatrix
     buildInfoKeys ++= Seq(
       smithy4sVersion
     ),
+    // todo: upstream should provide this
```

Comment:
@baccata shall we provide this in smithy4s in an easier way? (the ability to add default settings to any configuration)

Reply:
Sure